pkg/bridgeclient/api_impl.go

```diff
@@ -32,7 +32,7 @@ func handleGetRegister(c *gin.Context) {
 	store := persistence.GetKeyStore()
 	locationID := store.LoadLocationID("")
```

Comment:
You could default this to * instead of ""

Reply:
i need the actual locationID though